### ISSUE TITLE
Animation while tree expansion is lagging

### ISSUE THREAD

**Issue by ratnam99** [500tech/angular-tree-component#586]
Hi, I have been using this library for almost a year. But now, sometimes the animation of tree expansion is lagging. So, What can I do to fix this. 

Thanks in advance.
![expand slow](https://user-images.githubusercontent.com/29368749/37509158-390a8116-291c-11e8-9b3c-dc8043a9abcf.gif)

**Comment by shmarlya** [comment]
+1, my animation start lagging on contract, also i've got horizontal scroll bar appear

**Comment by adamkleingit** [comment]
Have you upgraded your version?
Are you using `animateExpand`, `animateSpeed`, and `animateAccelaration`?

**Comment by ratnam99** [comment]
Yes, I'm using `animateExpand`, `animateSpeed`, and `animateAccelaration`

**Comment by adamkleingit** [comment]
Can you reproduce in the plunkr:
https://plnkr.co/edit/u5fMmaM14rJ0aqG3k240?p=preview
?

**Comment by ratnam99** [comment]
I can provide you values of `animateExpand`, `animateSpeed`, and `animateAccelaration`

**Comment by ratnam99** [comment]
I was not able to reproduce it. Animation is working fine in the plunkr:
https://plnkr.co/edit/u5fMmaM14rJ0aqG3k240?p=preview

**Comment by ratnam99** [comment]
This is the modified plunkr code of `src/app-child-component.ts`
`import {Component, OnInit} from '@angular/core
import { TreeNode, TreeModel, TREE_ACTIONS, KEYS, IActionMapping, ITreeOptions } from 'angular-tree-component';

const actionMapping: IActionMapping = {
    mouse: {
        dblClick: (tree, node, $event) => {
            if (node.hasChildren) {
                TREE_ACTIONS.TOGGLE_EXPANDED(tree, node, $event);
            }
        },
        click: (tree, node, $event) => {
            $event.shiftKey ? TREE_ACTIONS.TOGGLE_SELECTED_MULTI(tree, node, $event) : TREE_ACTIONS.TOGGLE_SELECTED(tree, node, $event);
        }
    }
}; 

@Component({
  selector: 'my-app',
  template: `
    <tree-root 
      [nodes]="nodes"
      [options]="options"
    ></tree-root>
  `,
}) export class ChildComponent implements OnInit {
  
  
  
  showTree = true;
  nodes = NODES; // declared after component...
  options = OPTIONS;

  ngOnInit() {
  }
  

}

const OPTIONS = {
    actionMapping,
    animateExpand: true,
    animateSpeed: 100,
    animateAcceleration: 1.2
};

const NODES = [
  {
    id:'1', 
    name:'Root', 
    children: [
      {
        id:'2',
        name:'First Child'
      },
      {
        id:'3',
        name:'Second Child'
      }
    ]
  },
  {
    id:'4', 
    name:'Root2', 
    children: [
      {
        id:'5',
        name:'First Child',
        children: [
          { id: '7', name: 'grandchild' }
        ]
      },
      {
        id:'6',
        name:'Second Child'
      }
    ]
  },
]`

**Comment by shmarlya** [comment]
@adamkleingit update to latest version fix animation bug, but now there some warnings in console:
![console](https://user-images.githubusercontent.com/13673854/37584319-11ca252e-2b66-11e8-8045-2646878b5901.png)
